app/views/pages/download.html.erb

```diff
@@ -6,10 +6,10 @@
     RubyGems is a package management framework for Ruby. Download the latest version here:
   </p>
   <div id="formats" class="download__formats">
-    <%= link_to "tgz", "http://production.cf.rubygems.org/rubygems/rubygems-#{version_number}.tgz", class: "download__format" %>
-    <%= link_to "zip", "http://production.cf.rubygems.org/rubygems/rubygems-#{version_number}.zip", class: "download__format" %>
-    <%= link_to "gem", "http://production.cf.rubygems.org/rubygems/rubygems-update-#{version_number}.gem", class: "download__format" %>
-    <%= link_to "git", "http://github.com/rubygems/rubygems", class: "download__format" %>
+    <%= link_to "tgz", "https://rubygems.global.ssl.fastly.net/rubygems/rubygems-#{version_number}.tgz", class: "download__format" %>
+    <%= link_to "zip", "https://rubygems.global.ssl.fastly.net/rubygems/rubygems-#{version_number}.zip", class: "download__format" %>
+    <%= link_to "gem", "https://rubygems.global.ssl.fastly.net/rubygems/rubygems-update-#{version_number}.gem", class: "download__format" %>
```

Comment:
I’m :+1: for using HTTPS everywhere but I’m concerned about hard-coding domains that we don’t own. Is that a concern of anyone else? I’m thinking about people who copy these URLs into their scripts. I would rather have canonical HTTPS rubygems.org URLs that 302 to Fastly (or whatever other CDNs we decide to use in the future).